packages/esm-patient-common-lib/src/empty-state/empty-state.scss

```diff
@@ -49,7 +49,12 @@
   border-bottom: 0.375rem solid var(--brand-03);
 }
 
+.layer {
+  height: 100%;
```

Comment:
I would be in favor of replacing all these `height: 100%` lines by just declaring it with the extension slot. Using child selectors as needed. See [Extensions > Styling](https://o3-dev.docs.openmrs.org/docs/extension-system#styling) in the docs (one of those parts of the docs that I hope one day more people catch on to).

Whether it is appropriate to do the same with `flex` and `flex-direction`, I leave to your discretion.

Reply:
```scss
.slot {
  height: 100%;

  > div {
    height: 100%;
  }
}
```

Reply:
Fixed with 935d3860ce9d894cd24270f1e14d36592cb5b7e3